dockerfiles/Dockerfile.oracle

```diff
@@ -1,9 +1,10 @@
-FROM ghcr.io/verrazzano/verrazzano-base:v1.0.0-20230327155846-4653b27 as builder-base
+FROM ghcr.io/verrazzano/verrazzano-base-ol9:v1.0.0-20230627070315-744eefa as builder-base
 
 RUN mkdir -p /fluent-bit/bin /fluent-bit/etc /fluent-bit/log
+RUN microdnf update
```

Comment:
The purpose of these `verrazzano-base` images is that you don't need to run a `microdnf update`, so this is unnecessary here I think

Reply:
But what If, we have updates on base images later.